libview/ev-view-presentation.c

```diff
@@ -751,18 +751,44 @@ ev_view_presentation_goto_entry_grab_focus (EvViewPresentation *pview)
 	send_focus_change (pview->goto_entry, TRUE);
 }
 
+static void
+monitor_get_dimesions (EvViewPresentation *pview,
+		       gint     *width,
+		       gint     *height)
+{
+	GdkDisplay  *display;
+	GdkWindow   *gdk_window;
+	GdkMonitor  *monitor;
+	GdkRectangle geometry;
+
+	*width = 0;
+	*height = 0;
+
+	display = gtk_widget_get_display (GTK_WIDGET (pview));
+	gdk_window = gtk_widget_get_window (GTK_WIDGET (pview));
+
+	if (gdk_window) {
+		monitor = gdk_display_get_monitor_at_window (display,
+							     gdk_window);
+		gdk_monitor_get_workarea (monitor, &geometry);
```

Comment:
Why not use [gdk-monitor-get-geometry](https://developer.gnome.org/gdk3/stable/GdkMonitor.html#gdk-monitor-get-geometry) to get full monitor geometry?

Reply:
I agree with @rbuj, `gdk_monitor_get_workarea` will just return the monitor geometry on Wayland, but it's inconsistent on multi-monitor setups on X11. I think using `gdk_monitor_get_geometry` makes more sense here.

Reply:
@vkareh @rbuj 
feel free to close a working PR and change code like you prefer.
I only wanted to backport one commit from evince.
It wasn't my intention to change this code here.
If you have the idea than change it for yourself.

Reply:
@vkareh @rbuj 
again, feel free to open an alternative PR with using `gdk_monitor_get_workarea`, if my changes are so bad and wrong.
I don't know to do it.
Otherwise i'd like to merge  this PR.

Reply:
@raveit65 - `gdk_monitor_get_geometry (monitor, &geometry);` and `gdk_monitor_get_workarea (monitor, &geometry);` got the same geometry since the presentation filled the screen.

Test:

Open a document with more than n pages, start the presentation, press n key (n is a digit).

![Captura de pantalla a 2019-08-19 12-37-56](https://user-images.githubusercontent.com/10171411/63259477-cc412500-c27e-11e9-8e4b-ceb26267a069.png)

![Captura de pantalla a 2019-08-19 12-40-47](https://user-images.githubusercontent.com/10171411/63259485-d06d4280-c27e-11e9-8238-6a81e30e26aa.png)